site/content/docs/11-deploying.md

```diff
@@ -2,7 +2,17 @@
 title: Deployment
 ---
 
-Sapper apps run anywhere that supports Node 8 or higher.
+Sapper apps run anywhere that supports Node 10 or higher.
+
+### Deploying from sapper build
+
+If you're deploying your app to a different machine than the one you're building on, you will need to copy your `package.json` over and install your production dependencies (required by `server.js`) using `--only=prod` (or `--only=production`):
```

Comment:
I think it's a bit prescriptive to say that the installation has to happen on the production machine. Also, it's a bit redundant to include the flags here since they're shown below. What about something like?

```suggestion
You will need to have the server's dependencies available on your production machine. You can either install the dependencies on a machine with an empty `node_module/` folder and copy that folder over or you can make the `package.json` available on the production machine and install the dependencies there. Production dependencies can be installed with:
```

Reply:
If the folder isn't empty to start then I imagine that you could have dev dependencies in there. I haven't actually tested to see if you run with the production flag whether it will remove those, but I'm assuming it'd leave them there. And yes, I accidentally left off an `s`, which should be included

Reply:
I'm also not sure if the production flag for `npm i` would remove dev packages or not, but I guess this is where `npm ci --prod` would be useful because it automatically removes preexisting `node_modules` folder before it begins its install.

Reply:
Ah, that's good to know. I guess in that case with the `npm ci --prod` command we could drop the part about starting with an empty folder

Reply:
Okay, I've added the changes here with a slight tweak. Resolving this in favor of the updated changes made